FreeRTOS/Source/portable/GCC/RISC-V/port.c

```diff
@@ -196,7 +196,9 @@ void *pvReturnSealer;
 		               __CHERI_CAP_PERMISSION_PERMIT_LOAD_CAPABILITY__);
 
 	pvReturnSealer = cheri_setaddress( pvAlmightyDataCap, SANDBOX_RETURN_OTYPE );
-	pxPortSandboxReturnFunc = cheri_seal( xPortSandboxReturn, pvReturnSealer );
+	pxPortSandboxReturnFunc = cheri_setaddress( pvAlmightyCodeCap, (ptraddr_t)xPortSandboxReturn );
+

```

Comment:
Please delete this line. If you want to break things up you can put one before `pxPortSandboxReturnData`, but don't interrupt `pxPortSandboxReturnFunc`'s calculations in the middle.

Reply:
ditto above